ui/StatusQ/src/StatusQ/Components/StatusMessageHeader.qml

```diff
@@ -78,7 +78,7 @@ Item {
             }
             TapHandler {
                 enabled: root.displayNameClickable
-                onSingleTapped: root.clicked(this)
```

Comment:
Ideally this `sender` parameter could be removed.

In `MessageView`, in `onSenderNameClicked` the `openProfileContextMenu` may be called with any reasonable parent.

Reply:
Yeah... there's a lot of `sender`s in this and the parent components; probably worth filing a separate issue